app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.controller.js

```diff
@@ -176,7 +177,19 @@ export default class ProjectScenesController {
         this.activeModal = this.$uibModal.open({
             component: 'rfPublishModal',
             resolve: {
-                project: () => this.project
+                project: () => this.project,
+                tileTemplateUrl: () => {
+                    let host = this.$location.host();
+                    let protocol = this.$location.protocol();
+
+                    let port = this.$location.port();
+                    let formattedPort = port !== 80 || port !== 443 ? ':' + port : '';
+                    return `${protocol}://${host}${formattedPort}` +
+                        `/tiles/${this.project.organizationId}` +
+                        `/rf_airflow-user` +
+                        `/project/${this.project.id}/{level}/{col}/{row}/` +
```

Comment:
Yeah, I think this needs to go through at least one more round of work probably and a little bit of design -- I'll create an issue to come back to